ibis/backends/base/sql/compiler/query_builder.py

```diff
@@ -98,7 +107,7 @@ def _format_in_memory_table(self, op):
             rows = " UNION ALL ".join(f"(SELECT {raw_row})" for raw_row in raw_rows)
             return f"({rows})"
 
-    def _format_table(self, op):
+    def _format_table(self, op, no_alias: bool = False):
```

Comment:
just a thought: `no_alias = False` is a double negative, which seems slightly confusing compared to just doing `alias = True/False`